source/chapters/configuration.rst

```diff
@@ -183,3 +207,304 @@ already use JACK, it is easiest to let Mixxx suspend PulseAudio and use ALSA.
+             Mixxx can not recursively scan folders for compatible music files.
+
+**Import playlists**
+  You can import existing :file:`m3u`, :file:`pls` , :file:`m3u8`, :file:`pls`
+  playlist files from products other than Mixxx, see :ref:`library-playlists`.
+
+**Import music from CDs**
+  Mixxx can not play music from Audio CDs. Convert the content to compatible
+  files in good quality and add them to the Mixxx library. See
+  `<https://en.wikipedia.org/wiki/Ripping>`_
+
+.. _djing-changing-music-directories:
+
+Changing music directories
+==========================
+
+.. versionadded:: 1.12
+   Handles multiple music library folders and adds an option to move them to
+   another location without data loss.
+
+You can manually add, relink, and remove Mixxx music directories in
+:menuselection:`Preferences --> Library`.
+
+**Add a new music directory**
+  Mixxx handles multiple music library folders. Click :guilabel:`Add` to
+  browse to a directory where your music is stored. Mixxx will watch this
+  directory and its subdirectories for new tracks.
+
+  If you add a directory that is already in your library, or you are currently
+  :ref:`rescanning your library <library-root>`, the operation is canceled.
+
+  Directories can also be added from the :ref:`Browse <library-browse>` sidebar
+  item inside the library.
+
+**Relink a existing music directory**
+  If an existing music directory is moved, Mixxx doesn't know where to find the
+  audio files in it. Click :guilabel:`Relink` to select the music directory
+  in its new location. This will re-establish the links to the audio files in
+  the Mixxx library.
+
+**Remove a music directory**
+  Click :guilabel:`Remove`, and Mixxx will no longer watch a directory and
+  its subdirectories for new tracks, and asks what would you like to do with the
+  tracks from these directories.
+
+  * Select :guilabel:`Hide Tracks` to hide all tracks from this directory and
+    subdirectories.
+  * Select :guilabel:`Delete Track Metadata` to delete all metadata for these
+    tracks from Mixxx permanently
+  * Select :guilabel:`Leave Tracks Unchanged` to leave the tracks unchanged in
+    your library.
+
+  Hiding tracks saves their metadata in case you re-add them in the future.
+
+  Metadata means all track details (artist, title, playcount, etc.) as well as
+  beatgrids, hotcues, and loops. This choice only affects the Mixxx library.
+  No files on disk will be changed or deleted.
+
+.. hint:: When changing music directories, you might want to run a library
+          rescan afterwards. Select :menuselection:`Library --> Rescan Library`
+          in the menu.
+
+.. _djing-bpm-detection:
+
+BPM and Beat Detection Preferences
+==================================
+
+.. sectionauthor::
+   T.Rafreider <trafreider@mixxx.org>
+   S.Brandt <s.brandt@mixxx.org>
+
+Previous versions of Mixxx were able to detect BPM values but unable to
```

Comment:
Remove this bit about previous versions of mixxx.  Who cares if we used to suck?